Makefile.include

```diff
@@ -650,10 +650,12 @@ endef
 # Check if setsid command is available on the system for debug target
 # This is not the case on MacOSX, so it must be built on the fly
 ifneq (,$(filter debug, $(MAKECMDGOALS)))
-  ifneq (0,$(shell which setsid 2>&1 > /dev/null ; echo $$?))
-    SETSID = $(RIOTTOOLS)/setsid/setsid
-    $(call target-export-variables,debug,$(SETSID))
-    DEBUGDEPS += $(SETSID)
+  ifeq (Darvin,$(shell uname))
```

Comment:
Is darvin a typo?, also `OS` already exists everywhere.

```suggestion
  ifeq ($(OS),Darwin)
```

Reply:
Then I move that check to the outermost, as I imagine that a simple string comparison is faster than a string-comparison with filter function over `MAKECMDGOALS`.